.travis.yml

```diff
@@ -3,7 +3,7 @@ services:
  - docker
 
 before_install:
- - docker pull neverpanic/mococrw-build-env
+ - docker pull fmonteiro93/mococrw-build-env
```

Comment:
Maybe we should open a new axis and use a matrix build here to build in both the OpenSSL 1.0 and the OpenSSL 1.1 container?

According to https://docs.travis-ci.com/user/environment-variables/, something along the lines of

```yaml
env:
  matrix:
    - CONTAINER="fmonteiro93/mococrw-build-env"
    - CONTAINER="neverpanic/mococrw-build-env"
```

and changing the docker pull as well as the docker run to use `$CONTAINER` should probably work.

Also, you'll also have to change the `docker run` statements below to use your container.

Reply:
Nevermind, just read that you don't plan on keeping 1.0 compatibility. In this case, you'll only have to adjust the `docker run` statements.

Reply:
And you'll either have to set up automatic builds on hub.docker.com for your new repository or send me a pull request on my repository, ideally for a new branch. I'd then merge the change and configure the automatic builds to build the branch and tag it accordingly.

Ideally, this would all be part of the bmwcarit org as well, but we haven't moved it there yet.

Reply:
Thanks for the input!! I discussed this with @garw and we agreed to leave the setup of Travis outside the scope of this PR and remove the TravisCI verification for the OpenSSL1.1 branch for now. This will for sure be added in the future.